src/commands/login/sso.ts

```diff
@@ -11,8 +10,11 @@ export default class LoginSSO extends AuthCommand {
 
     public async run(): Promise<void> {
         const ssoAuth = new SSOAuth(this.writer)
-        this.token = await ssoAuth.getAccessToken()
-        storeAccessToken(this.token, this.authPath)
+        const accessToken = await ssoAuth.getAccessToken()
+        this.token = accessToken
```

Comment:
is this being used anywhere, I see that `authConfig.sso.accessToken` is being used instead

Reply:
All of the other commands set and use this.token, and this ensures that bugs will never be introduced in the case that someone assumes that this is true for all commands.